src/components/NodeEditor/NodeEditor.tsx

```diff
@@ -258,14 +437,41 @@ export default class NodeEditor extends React.PureComponent<NodeEditorProps, Nod
         }, []);
     }
 
-    private saveLocalizedExits(widgets: { [name: string]: any }): void {
-        const exits = this.getLocalizedExits(widgets);
-        const { iso } = this.props.language;
+    private saveLocalizations(widgets: { [name: string]: any }, cases?: CaseElementProps[]): void {
+        const updates = [...this.getLocalizedExits(widgets)];
+
+        if (cases && cases.length) {
+            updates.push(...this.getLocalizedCases(widgets));
+        }
+
+        this.updateLocalizations(this.props.language.iso, updates);
+    }
+
+    private getLocalizedCases(widgets: {
```

Comment:
moved this up from `SwitchRouter` as well because `saveLocalizations` relies on it